airbyte-local.sh

```diff
@@ -535,11 +873,13 @@ main() {
     trap cleanup SIGINT
     echo "Created folder $tempdir for temporary Airbyte files"
 
-    if ((no_src_pull)); then
-        log "Skipping pull of source image $src_docker_image"
-    else
-        log "Pulling source image $src_docker_image"
-        docker pull $src_docker_image
+    if [[ -z ${k8s_deployment+x} ]]; then
+        if ((no_src_pull)); then
+            log "Skipping pull of source image $src_docker_image"
+        else
+            log "Pulling source image $src_docker_image"
+            docker pull $src_docker_image
+        fi
```

Comment:
Why do you need to wrap this if you are making sure that `no_src_pull=1` [here ](https://github.com/faros-ai/airbyte-local-cli/pull/77/files#diff-1a4563128a0ebe9560b0903999361507964a6cf2b20950c1b4a859213b3cd9f0R286-R287) when k8s_deployment?

Reply:
Yes - I had first `no_src_pull=1` - which makes everything functionally correct as you noted - but then saw the log message printing in k8s deployment - which is confusing for the user (i.e., I am deploying to k8s but see - skipping pull of source image :)